content/en/logs/log_collection/kotlin-multiplatform.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Kotlin Multiplatform Log Collection
+description: Collect logs from your Kotlin Multiplatform applications.
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-kotlin-multiplatform
+  tag: "Source Code"
+  text: dd-sdk-kotlin-multiplatform Source code
+- link: logs/explorer
+  tag: Documentation
+  text: Learn how to explore your logs
+---
+
+## Overview
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Kotlin Multiplatform Monitoring is in private beta.
```

Comment:
```suggestion
Kotlin Multiplatform Monitoring is in preview.
```

We should change this to comply with the new [beta policy](https://datadoghq.atlassian.net/wiki/spaces/PM/pages/3310617880/Updated+Product+Lifecycle+Datadog)